Helper-BCC.lua

```diff
@@ -0,0 +1,39 @@
+local _, addon = ...
+local L = addon.L
+
+local intendedWoWProject = WOW_PROJECT_BURNING_CRUSADE_CLASSIC
+
+function addon:IsClassic()
```

Comment:
This part can be moved to to init option. We can load constants / spells based on set version

Reply:
It could be - I thought we could build on a "Helper" class that held a lot of the non-addon specific functions instead of adding them to the existing files and maybe drag out some functions to the Helper file.

Reply:
Ok